vips/vip-280/bscmainnet.ts

```diff
@@ -0,0 +1,48 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const XVS_VAULT_TREASURY = "0x269ff7818DB317f60E386D2be0B259e1a324a40a";
+export const VTREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+export const XVS = "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63";
+export const XVS_VAULT = "0x051100480289e704d20e9DB4804837068f3f9204";
+
+export const XVS_FOR_XVS_STORE = parseUnits("72905.06", 18);
+export const XVS_FOR_V_TREASURY = parseUnits("21789.17", 18);
+export const XVS_DISTRIBUTION_SPEED = parseUnits("0.063888888888888888", 18); // 1,840 XVS/day
+
+export const vip280 = () => {
```

Comment:
```suggestion
const vip280 = () => {
```
we can remove export in all vips as they are being exported at the end.